### PR TITLE
Case study week 8

### PR DESCRIPTION
**Pull request by vbrajvong**
http://vbrajvong.github.io/ptCaseStudy

### CODE REVIEW

**Comment by vbrajvong** [comment]
http://vbrajvong.github.io/ptCaseStudy/

File: activities.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+content="width=device-width, 
+initial-scale=1.0">
+</head>
+<div id="wrapper">
+    <header>
+        <h1>Pacific Trails Resort</h1>
+    </header>
+<nav>
+        <ul>
+            <li><a href="index.html">Home</a></li>
```
Comment (mbMosman) [inline activities.html]: Nice work converting the nav

**Comment by mbMosman** [comment]
I'm going to grade part 3 (Ch 7) with your updated files because they are better than the earlier submission.  Bonus for you since I'm grading late.  

However you still have validation errors which are repeatedly hurting your grade.  You need to get these corrected.

File: pacific.css [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+td, th { border: 1px solid #3399CC; padding: 5px; }
+
+td {text-align: center;}
+
+#text {text-align: left;}
+#text2 {text-align: left;}
+#text3 {text-align: left;}
+
+
+:nth-of-type {background-color:#F5FAFC}
```
Comment (mbMosman) [inline pacific.css]: This is also a validation error.  The pseudo class must be applied to a tag that is placed before the : 

See page 297 #5 for an example.

File: yurts.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+</tr>
+<tr>
+<td>Weekend Escape</td>
+<td id="text">Two breakfasts, a trail map, and a picnic snack</td>
+<td>2</td>
+<td>$450</td>
+</tr>
+<tr>
+ <td>Zen Retreat</td>
+<td id="text2">Four breakfasts, a trail map, a picnic snack, and a pass for the daily sunrise Yoga session</td>
```
Comment (mbMosman) [inline yurts.html]: The same style class of text can be applied to all 3 table cells.

File: pacific.css [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    font-size: 90%;
+}
+
+table { margin: auto; border: 1px solid #3399CC; width: 80%; border-spacing: 0; }
+
+td, th { border: 1px solid #3399CC; padding: 5px; }
+
+td {text-align: center;}
+
+#text {text-align: left;}
```
Comment (mbMosman) [inline pacific.css]: This could be done with one class, instead of 3 id rules.

File: pacific.css [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+}
+
+nav a:hover {
+    color: #333399;
+    background-color: #FFFFFF;
+}
+
+main {
+    background-color: #FFFFFF;
+    margin-left: 160px;
```
Comment (mbMosman) [inline pacific.css]: The padding is missing for the main content.

padding: 1px 20px 20px 30px;

File: pacific.css [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+        padding-bottom: 0.1em;
+        padding-right: 1em;
+        font-size: 90%;
+    }
+    h1 {font-size: 1.5em}
+    nav {padding: 0;}
+    nav li { display: block;
+    margin: 0;
+    border-bottom-style: solid;
+    border-bottom-color: #330000;}
```
Comment (mbMosman) [inline pacific.css]: The border size is missing so you get the default 3px instead of the requested 2px.

**Comment by mbMosman** [comment]
This is better, but make sure to validate the CSS too!

**Pull request closed without merging**